api-swagger-patched.yaml

```diff
@@ -42,6 +42,32 @@ definitions:
       data:
         $ref: "#/definitions/HealthCheck"
 
+  Provider:
+    type: string
+    x-ms-enum:
+      name: "Provider"
+    enum:
+      - manual
+      - bb
+      - bbe
+      - gh
+      - ghe
+      - gl
+      - gle
+
+  AccountProvider:
```

Comment:
Isn't this AccountIntegrations? I find weird having google as a "provider" 🤔Am I thinking wrong here?

Reply:
I believe this maps better on the codacy-website than here. We may change it but it will have to go through on website first.

Reply:
Ok I guess we can discuss this as an improvement (or not, my thought can be wrong here) later on